src/extension/providers/package-reference-completion.ts

```diff
@@ -55,20 +55,21 @@ export class PackageReferenceCompletionProvider implements vscode.CompletionItem
 
     /**
      * Provide completion items for the specified document position.
-     * 
+     *
      * @param document The target document.
      * @param position The position within the target document.
      * @param token A vscode.CancellationToken that can be used to cancel completion.
-     * 
+     *
      * @returns {Promise<vscode.CompletionList>} A promise that resolves to the completion item list.
      */
     private async provideCompletionItemsCore(document: vscode.TextDocument, position: vscode.Position, token: vscode.CancellationToken): Promise<vscode.CompletionList> {
         const completionItems: vscode.CompletionItem[] = [];
-        
+
         // To keep things simple, we only parse one line at a time (if your PackageReference element spans more than one line, too bad).
         const line = document.lineAt(position);
         const xml = this.parser.parseFromString(line.text);
-        const elements = xml.getElementsByTagName('PackageReference');
+        const elements = Array.from(xml.getElementsByTagName('PackageReference'))
+            .concat(Array.from(xml.getElementsByTagName('GlobalPackageReference')));
```

Comment:
This is a special case from https://github.com/microsoft/MSBuildSdks/tree/master/src/CentralPackageVersions